deployments/prob140/image/requirements.txt

```diff
@@ -0,0 +1,25 @@
+#
+# data8; foundation
+tornado==4.5.3
+notebook==5.6.0
+datascience==0.10.4
+matplotlib==2.0.0
+numpy==1.14.5
+pandas==0.19.2
+scipy==0.19.1
+statsmodels==0.8.0
+nbgitpuller==0.6.1
+nbzip==0.0.4
+jupyterlab==0.33.11
+ipywidgets==7.2.1
+jupyterhub==0.8.1
+appmode==0.4.0
+nbresuse==0.3.0
+#
+# r
+nbserverproxy==0.8.3
```

Comment:
we don't have rstudio here, does prob140 use R? if not we should remove it.

Reply:
Fixed! Thanks, I'd removed the R bits from the user image but missed these.